packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -29,7 +33,7 @@ import { L1StandardBridge } from "src/L1/L1StandardBridge.sol";
 import { OptimismMintableERC20Factory } from "src/universal/OptimismMintableERC20Factory.sol";
 
 /// @custom:proxied true
-contract OPStackManager is ISemver {
+contract OPStackManager is ISemver, Initializable {
```

Comment:
Might want to remove this based on decision made between initialize vs setRelease.

Reply:
Not using set release, so keeping this.